libcxx/test/std/utilities/format/format.arguments/format.args/get.pass.cpp

```diff
@@ -82,11 +82,6 @@ void test_string_view(From value) {
 template <class CharT>
 void test() {
   using Context = std::basic_format_context<CharT*, CharT>;
-  {
-    const std::basic_format_args<Context> format_args{};
-    ASSERT_NOEXCEPT(format_args.get(0));
-    assert(!format_args.get(0));
-  }
 
   using char_type = typename Context::char_type;
```

Comment:
Let's remove the blank line since it looks odd now.
```suggestion
  using char_type = typename Context::char_type;
```